### ISSUE TITLE
Guidance for Kubernetes Deploy?

### ISSUE THREAD

**Issue by zchef2k** [open-eats/OpenEats#118]
App looks great. I'm looking to deploy this in my rancher lab and am having a bit of problem jumping off. Any chance there's a helm chart anywhere? Any guidance is appreciated.

**Comment by zchef2k** [comment]
None needed. I've mostly got this up and going. Struggling with proxying api content but I'll open a separate issue for that.

**Comment by mitswan** [comment]
Mind sharing the chart?

**Comment by zchef2k** [comment]
> Mind sharing the chart?

I never created a chart. I ended up deploying the web and api components manually and a third apache container to act as the proxy. All three make claim to the same PV in some form or fashion to access common files.

I did this back in April and it was somewhat of a struggle, mostly for me learning django and this app's architecture. I didn't take the best notes when I got it running but am happy to replay what I did in case it helps you.
